index.html

```diff
@@ -13,7 +13,25 @@
 </head>
 
 <body>
+  <header class="cabecario">  
```

Comment:
Colocar uma classe irmã. 
```suggestion
  <header class="cabecario organizacao-cabecalho">  
```